content/topics/bridging-course-hard-skills/maths-for-programmers/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Learning How To Learn
+ready: True
+---
+
+### Maths for Programmers
+Understanding these foundational mathematical concepts will help you a lot with your coding abilities:
+
+Khan academy has a few courses that are relevant:
```

Comment:
This page wont look good when it's rendered.

Please run hugo and make sure that you look t every single page that you create. The same problem exists elsewhere in this PR. You should be able to spot it if you just look

Reply:
And these should probably be seperte pages anyway - don't we want to track each task individually?